modules/services/aerospace/default.nix

```diff
@@ -0,0 +1,156 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+with lib;
+
+let
+  cfg = config.services.aerospace;
+
+  format = pkgs.formats.toml { };
+  configFile = format.generate "aerospace.toml" cfg.settings;
+in
+
+{
+  options = with types; {
+    services.aerospace = {
+      enable = mkEnableOption "AeroSpace window manager";
+
+      package = mkOption {
+        type = types.path;
+        default = pkgs.aerospace;
+        description = "The AeroSpace package to use.";
+      };
+
+      settings = mkOption {
+        type = submodule {
+          freeformType = format.type;
+          options = {
+            start-at-login = mkOption {
+              type = addCheck bool (b: !false || !cfg.enable);
```

Comment:
Doesn't that function always equal `true`?

Reply:
Instead of implementing this with `addCheck`

I think it would make more sense to implement with an assertion